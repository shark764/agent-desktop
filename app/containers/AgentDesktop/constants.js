/*
 *
 * AgentDesktop constants
 *
 */

export const SET_USER_CONFIG = 'app/AgentDesktop/SET_USER_CONFIG';
export const SET_EXTENSIONS = 'app/AgentDesktop/SET_EXTENSIONS';
export const SET_ACTIVE_EXTENSION = 'app/AgentDesktop/SET_ACTIVE_EXTENSION';
export const SET_QUEUES = 'app/AgentDesktop/SET_QUEUES';
export const SET_PRESENCE = 'app/AgentDesktop/SET_PRESENCE';
export const GO_NOT_READY = 'app/AgentDesktop/GO_NOT_READY';
export const UPDATE_WRAPUP_DETAILS = 'app/AgentDesktop/UPDATE_WRAPUP_DETAILS';
export const ADD_SCRIPT = 'app/AgentDesktop/ADD_SCRIPT';
export const REMOVE_SCRIPT = 'app/AgentDesktop/REMOVE_SCRIPT';
export const SET_INTERACTION_STATUS = 'app/AgentDesktop/SET_INTERACTION_STATUS';
export const START_OUTBOUND_INTERACTION = 'app/AgentDesktop/START_OUTBOUND_INTERACTION';
export const INITIALIZE_OUTBOUND_SMS = 'app/AgentDesktop/INITIALIZE_OUTBOUND_SMS';
export const ADD_INTERACTION = 'app/AgentDesktop/ADD_INTERACTION';
export const WORK_INITIATED = 'app/AgentDesktop/WORK_INITIATED';
export const REMOVE_INTERACTION = 'app/AgentDesktop/REMOVE_INTERACTION';
export const SET_INTERACTION_QUERY = 'app/AgentDesktop/SET_INTERACTION_QUERY';
export const ADD_SEARCH_FILTER = 'app/AgentDesktop/ADD_SEARCH_FILTER';
export const REMOVE_SEARCH_FILTER = 'app/AgentDesktop/REMOVE_SEARCH_FILTER';
export const SET_MESSAGE_HISTORY = 'app/AgentDesktop/SET_MESSAGE_HISTORY';
export const SET_CONTACT_ACTION = 'app/AgentDesktop/SET_CONTACT_ACTION';
export const ASSIGN_CONTACT = 'app/AgentDesktop/ASSIGN_CONTACT';
export const SET_SIDE_PANEL_TAB_INDEX = 'app/AgentDesktop/SET_SIDE_PANEL_TAB_INDEX';
export const LOAD_CONTACT_INTERACTION_HISTORY = 'app/AgentDesktop/LOAD_CONTACT_INTERACTION_HISTORY';
export const SET_CONTACT_INTERACTION_HISTORY = 'app/AgentDesktop/SET_CONTACT_INTERACTION_HISTORY';
export const SET_CONTACT_HISTORY_INTERACTION_DETAILS_LOADING = 'app/AgentDesktop/SET_CONTACT_HISTORY_INTERACTION_DETAILS_LOADING';
export const SET_CONTACT_HISTORY_INTERACTION_DETAILS = 'app/AgentDesktop/SET_CONTACT_HISTORY_INTERACTION_DETAILS';
export const LOAD_HISTORICAL_INTERACTION_BODY = 'app/AgentDesktop/LOAD_HISTORICAL_INTERACTION_BODY';
export const UPDATE_CONTACT_HISTORY_INTERACTION_DETAILS = 'app/AgentDesktop/UPDATE_CONTACT_HISTORY_INTERACTION_DETAILS';
export const ADD_NOTES_TO_CONTACT_INTERACTION_HISTORY = 'app/AgentDesktop/ADD_NOTES_TO_CONTACT_INTERACTION_HISTORY';
export const UPDATE_CONTACT = 'app/AgentDesktop/UPDATE_CONTACT';
export const DELETE_CONTACTS = 'app/AgentDesktop/DELETE_CONTACTS';
export const SELECT_CONTACT = 'app/AgentDesktop/SELECT_CONTACT';
export const REMOVE_CONTACT = 'app/AgentDesktop/REMOVE_CONTACT';
export const ADD_MESSAGE = 'app/AgentDesktop/ADD_MESSAGE';
export const SELECT_INTERACTION = 'app/AgentDesktop/SELECT_INTERACTION';
export const SET_CUSTOM_FIELDS = 'app/AgentDesktop/SET_CUSTOM_FIELDS';
export const SET_EMAIL_PLAIN_BODY = 'app/AgentDesktop/SET_EMAIL_PLAIN_BODY';
export const SET_EMAIL_HTML_BODY = 'app/AgentDesktop/SET_EMAIL_HTML_BODY';
export const SET_EMAIL_DETAILS = 'app/AgentDesktop/SET_EMAIL_DETAILS';
export const SET_EMAIL_ATTACHMENT_URL = 'app/AgentDesktop/SET_EMAIL_ATTACHMENT_URL';
export const START_WARM_TRANSFERRING = 'app/AgentDesktop/START_WARM_TRANSFERRING';
export const TRANSFER_CANCELLED = 'app/AgentDesktop/TRANSFER_CANCELLED';
export const RESOURCE_ADDED = 'app/AgentDesktop/RESOURCE_ADDED';
export const UPDATE_RESOURCE_NAME = 'app/AgentDesktop/UPDATE_RESOURCE_NAME';
export const UPDATE_RESOURCE_STATUS = 'app/AgentDesktop/UPDATE_RESOURCE_STATUS';
export const HOLD_ME = 'app/AgentDesktop/HOLD_ME';
export const RESUME_ME = 'app/AgentDesktop/RESUME_ME';
export const RESOURCE_REMOVED = 'app/AgentDesktop/RESOURCE_REMOVED';
export const MUTE_CALL = 'app/AgentDesktop/MUTE_CALL';
export const UNMUTE_CALL = 'app/AgentDesktop/UNMUTE_CALL';
export const HOLD_CALL = 'app/AgentDesktop/HOLD_CALL';
export const RESUME_CALL = 'app/AgentDesktop/RESUME_CALL';
export const RECORD_CALL = 'app/AgentDesktop/RECORD_CALL';
export const STOP_RECORD_CALL = 'app/AgentDesktop/STOP_RECORD_CALL';
export const EMAIL_CREATE_REPLY = 'app/AgentDesktop/EMAIL_CREATE_REPLY';
export const EMAIL_ADD_ATTACHMENT = 'app/AgentDesktop/EMAIL_ADD_ATTACHMENT';
export const EMAIL_REMOVE_ATTACHMENT = 'app/AgentDesktop/EMAIL_REMOVE_ATTACHMENT';
export const EMAIL_UPDATE_REPLY = 'app/AgentDesktop/EMAIL_UPDATE_REPLY';
export const EMAIL_CANCEL_REPLY = 'app/AgentDesktop/EMAIL_CANCEL_REPLY';
export const EMAIL_SEND_REPLY = 'app/AgentDesktop/EMAIL_SEND_REPLY';
export const UPDATE_NOTE = 'app/AgentDesktop/UPDATE_NOTE';
export const UPDATE_SCRIPT_VALUES = 'app/AgentDesktop/UPDATE_SCRIPT_VALUES';
export const SET_DISPOSITION_DETAILS = 'app/AgentDesktop/SET_DISPOSITION_DETAILS';
export const SELECT_DISPOSITION = 'app/AgentDesktop/SELECT_DISPOSITION';
export const SHOW_REFRESH_NOTIF = 'app/AgentDesktop/SHOW_REFRESH_NOTIF';

export const DEFAULT_LOCALE = 'en-US';
