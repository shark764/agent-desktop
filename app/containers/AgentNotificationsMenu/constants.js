/*
 * Copyright Â© 2015-2017 Serenova, LLC. All rights reserved.
 */

/*
 *
 * AgentNotificationsMenu constants
 *
 */

// Reducer constants
export const SET_AUDIO_NOTIFICATIONS_PREFERENCE =
  'app/AgentNotificationsMenu/SET_AUDIO_NOTIFICATIONS_PREFERENCE';
export const SET_VISUAL_NOTIFICATIONS_PREFERENCE =
  'app/AgentNotificationsMenu/SET_VISUAL_NOTIFICATIONS_PREFERENCE';

// Saga constants
export const INITIALIZE_NOTIFICATION_PREFERENCES =
  'app/AgentNotificationsMenu/INITIALIZE_NOTIFICATION_PREFERENCES';
export const TOGGLE_AUDIO_NOTIFICATIONS_PREFERENCE =
  'app/AgentNotificationsMenu/TOGGLE_AUDIO_NOTIFICATIONS_PREFERENCE';
export const TOGGLE_VISUAL_NOTIFICATIONS_PREFERENCE =
  'app/AgentNotificationsMenu/TOGGLE_VISUAL_NOTIFICATIONS_PREFERENCE';
