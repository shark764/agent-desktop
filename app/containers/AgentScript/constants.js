/*
 * Copyright Â© 2015-2017 Serenova, LLC. All rights reserved.
 */

/*
 *
 * AgentScript constants
 *
 */

export const SEND_SCRIPT = 'app/AgentScript/SEND_SCRIPT';
