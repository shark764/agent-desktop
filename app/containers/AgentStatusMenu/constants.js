/*
 *
 * AgentStatusMenu constants
 *
 */

export const DEFAULT_ACTION = 'app/AgentStatusMenu/DEFAULT_ACTION';
