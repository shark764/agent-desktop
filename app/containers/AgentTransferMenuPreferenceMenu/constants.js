/*
 * Copyright Â© 2015-2017 Serenova, LLC. All rights reserved.
 */

/*
 *
 * AgentTranferMenu constants
 *
 */

// Redux constants
export const SET_AGENTS_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/SET_AGENTS_TRANSFER_MENU_PREFERENCE';

//  Redux Queues Constants
export const SET_SHOW_QUEUES_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/SET_SHOW_QUEUES_TRANSFER_MENU_PREFERENCE';
export const TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE';
export const TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE';

//  Redux Transfer Lists Constants
export const SET_SHOW_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/SET_SHOW_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE';
export const TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE';
export const TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE';

// Saga Constants
export const INITIALIZE_TRANSFER_MENU_PREFERENCES =
  'app/AgentTransferMenuPreferenceMenu/INITIALIZE_TRANSFER_MENU_PREFERENCES';
export const TOGGLE_AGENTS_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/TOGGLE_AGENTS_TRANSFER_MENU_PREFERENCE';
export const TOGGLE_SHOW_QUEUES_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/TOGGLE_SHOW_QUEUES_TRANSFER_MENU_PREFERENCE';
export const TOGGLE_SHOW_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE =
  'app/AgentTransferMenuPreferenceMenu/TOGGLE_SHOW_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE';
