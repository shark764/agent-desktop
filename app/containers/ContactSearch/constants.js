/*
 * Copyright Â© 2015-2017 Serenova, LLC. All rights reserved.
 */

/*
 *
 * ContactSearch constants
 *
 */

export const SEARCH_CONTACTS = 'app/SearchContacts/SEARCH_CONTACTS';
