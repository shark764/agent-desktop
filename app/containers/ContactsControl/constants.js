/*
 *
 * ContactsControl constants
 *
 */

export const CLEAR_SEARCH_RESULTS = 'app/ContactsControl/CLEAR-SEARCH_RESULTS';
export const SET_SEARCH_RESULTS = 'app/ContactsControl/SET_SEARCH_RESULTS';
