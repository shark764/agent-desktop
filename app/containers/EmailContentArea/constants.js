/*
 * Copyright Â© 2015-2017 Serenova, LLC. All rights reserved.
 */

/*
 *
 * EmailContentArea constants
 *
 */

export const START_OUTBOUND_EMAIL = 'app/EmailContentArea/START_OUTBOUND_EMAIL';
