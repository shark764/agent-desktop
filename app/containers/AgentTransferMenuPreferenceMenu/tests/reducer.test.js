import { fromJS } from 'immutable';
import transferMenuPreferencesReducer from '../reducer';
import * as constant from '../constants';

describe('agentTransferMenuPreferenceMenuReducer', () => {
  it('returns the correct initial state', () => {
    expect(transferMenuPreferencesReducer(undefined, {})).toMatchSnapshot();
  });

  let action;
  let initialState;

  const runReducerAndExpectSnapshot = () => {
    expect(
      transferMenuPreferencesReducer(fromJS(initialState), action)
    ).toMatchSnapshot();
  };

  describe('SET_AGENTS_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        agentsTransferMenu: false,
      };
      action = {
        type: constant.SET_AGENTS_TRANSFER_MENU_PREFERENCE,
        agentsTransferMenu: false,
      };
    });
    it('changes agentsTransferMenu preference', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('SET_SHOW_QUEUES_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        transferMenuPreferences: {
          showQueues: false,
        },
      };
      action = {
        type: constant.SET_SHOW_QUEUES_TRANSFER_MENU_PREFERENCE,
        showQueues: true,
      };
    });
    it('opens queues dropdown', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedQueues: [],
      };
      action = {
        queue: '123',
        type: constant.TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle one queue to selected list', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedQueues: ['123', 'abc'],
      };
      action = {
        queue: '123',
        type: constant.TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle one queue and unselect it', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedQueues: [],
      };
      action = {
        queues: ['123', 'abc', '3d2'],
        type: constant.TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle all queues to selected list', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedQueues: ['123', 'abc', '3d2'],
      };
      action = {
        queues: [],
        type: constant.TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle all queues and return empty list', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedQueues: ['abc'],
      };
      action = {
        queues: ['123'],
        type: constant.TOGGLE_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle all stored queues to selected list', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('SET_SHOW_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        showTransferLists: false,
      };
      action = {
        type: constant.SET_SHOW_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE,
        showTransferLists: true,
      };
    });
    it('opens transfer list dropdown', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedTransferLists: [],
      };
      action = {
        transferList: '123',
        type: constant.TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle one transfer list to selected list', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedTransferLists: ['123'],
      };
      action = {
        transferList: '123',
        type: constant.TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle one transfer list and unselect it', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedTransferLists: ['123'],
      };
      action = {
        transferLists: ['abc', '3d2'],
        type:
          constant.TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle all transfer lists to selected list', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedTransferLists: ['123', 'abc', '3d2'],
      };
      action = {
        transferLists: [],
        type:
          constant.TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle all transfer lists and unselect it', () => {
      runReducerAndExpectSnapshot();
    });
  });

  describe('TOGGLE_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE', () => {
    beforeEach(() => {
      initialState = {
        selectedTransferLists: ['asd'],
      };
      action = {
        transferLists: ['321'],
        type: constant.TOGGLE_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE,
      };
    });
    it('toggle all stored transfer lists to selected list', () => {
      runReducerAndExpectSnapshot();
    });
  });
});
