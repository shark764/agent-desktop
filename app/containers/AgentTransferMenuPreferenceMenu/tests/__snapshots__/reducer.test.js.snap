// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`agentTransferMenuPreferenceMenuReducer SET_AGENTS_TRANSFER_MENU_PREFERENCE changes agentsTransferMenu preference 1`] = `
Immutable.Map {
  "agentsTransferMenu": false,
}
`;

exports[`agentTransferMenuPreferenceMenuReducer SET_SHOW_QUEUES_TRANSFER_MENU_PREFERENCE opens queues dropdown 1`] = `
Immutable.Map {
  "transferMenuPreferences": Immutable.Map {
    "showQueues": false,
  },
  "showQueues": true,
}
`;

exports[`agentTransferMenuPreferenceMenuReducer SET_SHOW_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE opens transfer list dropdown 1`] = `
Immutable.Map {
  "showTransferLists": true,
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE toggle all queues and return empty list 1`] = `
Immutable.Map {
  "selectedQueues": Immutable.List [],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_ALL_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE toggle all queues to selected list 1`] = `
Immutable.Map {
  "selectedQueues": Immutable.List [
    "123",
    "abc",
    "3d2",
  ],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE toggle all transfer lists and unselect it 1`] = `
Immutable.Map {
  "selectedTransferLists": Immutable.List [],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_ALL_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE toggle all transfer lists to selected list 1`] = `
Immutable.Map {
  "selectedTransferLists": Immutable.List [
    "123",
    "abc",
    "3d2",
  ],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE toggle one queue and unselect it 1`] = `
Immutable.Map {
  "selectedQueues": Immutable.List [
    "abc",
  ],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_SELECTED_QUEUE_TRANSFER_MENU_PREFERENCE toggle one queue to selected list 1`] = `
Immutable.Map {
  "selectedQueues": Immutable.List [
    "123",
  ],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_SELECTED_QUEUES_TRANSFER_MENU_PREFERENCE toggle all stored queues to selected list 1`] = `
Immutable.Map {
  "selectedQueues": Immutable.List [
    "123",
  ],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE toggle one transfer list and unselect it 1`] = `
Immutable.Map {
  "selectedTransferLists": Immutable.List [],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_SELECTED_TRANSFER_LIST_TRANSFER_MENU_PREFERENCE toggle one transfer list to selected list 1`] = `
Immutable.Map {
  "selectedTransferLists": Immutable.List [
    "123",
  ],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer TOGGLE_SELECTED_TRANSFER_LISTS_TRANSFER_MENU_PREFERENCE toggle all stored transfer lists to selected list 1`] = `
Immutable.Map {
  "selectedTransferLists": Immutable.List [
    "321",
  ],
}
`;

exports[`agentTransferMenuPreferenceMenuReducer returns the correct initial state 1`] = `
Immutable.Map {
  "agentsTransferMenu": false,
  "showQueues": false,
  "showTransferLists": false,
  "selectedQueues": Immutable.List [],
  "selectedTransferLists": Immutable.List [],
}
`;
